miso-web/src/main/webapp/scripts/form_indexfamily.js

```diff
@@ -0,0 +1,119 @@
+if (typeof FormTarget === 'undefined') {
+  FormTarget = {};
```

Comment:
Could replace this block with `FormTarget = FormTarget || {};`

Reply:
I feel like there was a reason I didn't use that syntax originally on all of the form scripts, but it doesn't seem to break anything, so :man_shrugging: